genie-web/src/main/java/com/netflix/genie/web/properties/LocalAgentLauncherProperties.java

```diff
@@ -109,4 +112,20 @@
      * Launch agent as the user in the job request (launches as the server user if false).
      */
     private boolean runAsUserEnabled;
+
+    /**
+     * How long after the job information for this host is written into a local cache is it evicted.
+     *
+     * Should be higher than {@link #getHostInfoRefreshAfter()}
+     */
+    @DurationUnit(ChronoUnit.SECONDS)
+    private Duration hostInfoExpireAfter = Duration.ofMinutes(1L);
+
+    /**
+     * How long after the job information for this host is written should it be automatically refreshed from
+     * the underlying data source.
+     *
+     * Should be lower than {@link #getHostInfoExpireAfter()}
+     */
+    private Duration hostInfoRefreshAfter = Duration.ofSeconds(30L);
```

Comment:
The one above is decorated with `@DurationUnit`, but this one is not

Reply:
Yeah. The `@DurationUnit` is only to tell spring what to consider a value without a qualifier like `s` or `m`. Anything not annotated the default is milliseconds. So in this case I just put it to `SECONDS` on the expiry one as another catch condition for people not to accidentally make expiry < refresh

Reply:
I think having these 2 right next to eachother with 2 different units is a little confusing.
Personally I'd leave everything in milliseconds which is "convention" in this project.

One larger than the other, means 100% cache miss, which is bad but not terrible.

A worst mistake that could happen here is:
`host-info-expire-after: 3000`

Reply:
Fair enough I'll remove the first one before merging this.